src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  Route,
+  Routes,
+  Navigate,
+} from "react-router-dom";
+import { AuthProvider } from "./contexts/AuthContext";
+import { Login } from "./pages/Login";
+import { ToDoApp } from "./pages/ToDoApp";
+import Navbar from "./components/common/ButtonAppBar";
+import { useLinks } from "./hooks/useLinks";
+
+const AppRoutes: React.FC = () => {
+  const { availableLinks } = useLinks();
+  const user = availableLinks.find(link => link.path === "/todo")
+
+  return (
+    <Routes>
+        <Route
```

Comment:
Create enum `RoutePolicy` (ForAnyone, ForGuests, ForAuthenticated) & `Router.tsx` to return list of routes according to RoutePolicy